README.md

```diff
@@ -1,54 +1,142 @@
+This repository provides a webhook to use cert-manager with Anexia CloudDNS.
 
-This is useful if you need to use cert-manager with a DNS provider that is not
-officially supported in cert-manager core.
+## Requirements
+-   [go](https://golang.org/) >= 1.13.0
+-   [helm](https://helm.sh/) >= v3.0.0
+-   [cert-manager](https://cert-manager.io/) >= 1.6.0
 
-## Why not in core?
+## Installation
 
-As the project & adoption has grown, there has been an influx of DNS provider
-pull requests to our core codebase. As this number has grown, the test matrix
-has become un-maintainable and so, it's not possible for us to certify that
-providers work to a sufficient level.
+### cert-manager
 
-By creating this 'interface' between cert-manager and DNS providers, we allow
-users to quickly iterate and test out new integrations, and then packaging
-those up themselves as 'extensions' to cert-manager.
+Follow the [instructions](https://cert-manager.io/docs/installation/) in the cert-manager documentation to install it within your cluster.
 
-We can also then provide a standardised 'testing framework', or set of
-conformance tests, which allow us to validate the a DNS provider works as
-expected.
+### Webhook
 
-## Creating your own webhook
+#### Using public helm chart
+```bash
+helm repo add cert-manager-webhook-anexia https://toothstone.github.io/cert-manager-webhook-anexia
+helm install --namespace cert-manager cert-manager-webhook-anexia cert-manager-webhook-anexia/cert-manager-webhook-anexia
+```
 
-Webhook's themselves are deployed as Kubernetes API services, in order to allow
-administrators to restrict access to webhooks with Kubernetes RBAC.
+#### From local checkout
 
-This is important, as otherwise it'd be possible for anyone with access to your
-webhook to complete ACME challenge validations and obtain certificates.
+```bash
+helm install --namespace cert-manager cert-manager-webhook-anexia deploy/cert-manager-webhook-anexia
+```
+**Note**: The kubernetes resources used to install the webhook should be deployed within the same namespace as the cert-manager.
 
-To make the set up of these webhook's easier, we provide a template repository
-that can be used to get started quickly.
+To uninstall the webhook run
+```bash
+helm uninstall --namespace cert-manager cert-manager-webhook-anexia
+```
+## Usage
+
+### Issuer
+
+Create a `ClusterIssuer` or `Issuer` resource as follows:
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: anexia-issuer
+spec:
+  acme:
+    email: mail@example.com # REPLACE THIS WITH YOUR EMAIL!!!
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: anexia-issuer-account-key
+    solvers:
+    - dns01:
+        webhook:
+          groupName: acme.anexia.com
+          solverName: anexia
+          config:
+            apiUrl: https://engine.anexia-it.com/api/clouddns/v1/zone.json # Don't change this if you're an Anexia customer
```

Comment:
maybe have a default value for this hard-coded and only document the possibility to set this, not having users to write the one valid value?

Reply:
I'm not really a fan of putting that URL into the binary, but from a UX point it is probably the sensible choice. I'll look into it.

Reply:
where are you even using this? The `go-anxcloud` handles URLs already, only optionally supporting a `BaseURL` to be configured as option (though this is not released yet)?

Reply:
Indeed, this is not used anywhere :facepalm: 
Probably an oversight when go-anxcloud was first introduced to the original code base :grimacing:

Reply:
Maybe keep around for the `BaseURL` option in the next `go-anxcloud` release ^^

Reply:
Once go-anxcloud is capable of having a configurable API URL (and if there is demand) we can reintroduce this, but for now we might as well drop it completely.